### DISCUSSION TITLE
run remote using renative+reactnative +tizentv,webostv

### DISCUSSION THREAD

**Discussion by aakashcdn** [pavjacko/renative#942] [Q&A]
when i run tizentv and webostv on simulator my tv remote not navigate input field how can it work.?

**Comment by pauliusguzas** [comment]
@aakashcdn thanks for using renative, so the issue is that tizentv and webos simulator remotes do not work? What versions of what packages are you using? Maybe the issue is with simulators themself, did you try just using remotes to navigate through tv simulator settings or some other apps, does it work there? Any information is appreciated to help you solve your issue faster
